src/widgets/NftDetailWidget/subcomponents/NftDetailPortrait/NftDetailPortrait.tsx

```diff
@@ -4,7 +4,7 @@ import './NftDetailPortrait.scss';
 
 interface CollectionPortraitProps {
   backgroundImage: string;
-  className?: string;
+  className: string;
```

Comment:
Why is it mandatory to have the className parameter?

Reply:
Because it has a default value so is never undefined.
We can make it optional, I don't mind :man_shrugging: